src/NuGetForUnity/Editor/InstalledPackagesManager.cs

```diff
@@ -256,38 +256,17 @@ internal static bool RemoveUnnecessaryPackages()
                 return false;
             }
 
-            var directories = Directory.GetDirectories(ConfigurationManager.NugetConfigFile.RepositoryPath, "*", SearchOption.TopDirectoryOnly);
```

Comment:
pleas adjust the documentation of the `RemoveUnnecessaryPackages` method as it now doesn't check for a directory.